lib/server.js

```diff
@@ -22,7 +22,7 @@ module.exports = function createServer(config, log) {
   if (config.ipBlocklist.enable) {
     var IPBlocklistManager = require('./ip_blocklist_manager')(log, config)
     var blockListManager = new IPBlocklistManager()
-    startupDefers.push(blockListManager.load(config.ipBlocklist.lists))
+    startupDefers.push(blockListManager.load([].concat(config.ipBlocklist.lists, config.ipBlocklist.logOnlyLists)))
```

Comment:
It seems convoluted to concat the two lists here, and then check `config.ipBlocklist.logOnlyLists` again to distinguish them when we're looking for hits.  I wonder if it'd be cleaner to pass the lists in separately and let the `IPBlocklistManager` class deal with them internally.  Something like:

```
  IPBlocklistManager.prototype.load = function (lists, logOnlyLists) {
    var self = this

    if (! Array.isArray(lists) || ! Array.isArray(logOnlyLists)) {
      throw Error('lists must be an array')
    }

    var loadedLists = lists.map(function (listPath) {
      var blocklist = new IPBlocklist()
      self.ipBlocklists.push(blocklist)
      return blocklist.load(listPath)
    })

    loadedLists = loadedLists.concat(logOnlyLists.map(function (listPath) {
      var blocklist = new IPBlocklist()
      blocklist.logOnly = true
      self.ipBlocklists.push(blocklist)
      return blocklist.load(listPath)
    })

    return Promise.all(loadedLists)
  }
```

In the `contains` method, you could then check the `logOnly` property of each list as you iterate through them, rather than doing a separate `logOnlyLists.indexOf` lookup.